activemq-jaas/src/main/java/org/apache/activemq/jaas/LDAPLoginModule.java

```diff
@@ -450,7 +440,6 @@ protected boolean bindUser(DirContext context, String dn, String password) throw
                 log.debug("User " + dn + " successfully bound.");
             }
         } catch (AuthenticationException e) {
-            isValid = false;
```

Comment:
Why is this being removed?

Reply:
Because it is `false` here and this assignment is not needed.
Logic hasn't been changed

Reply:
I agree with @Dm-Chebotarskyi : `isValid` is already set to `false` at line 429.